test/Clang/clang_static_library.py

```diff
@@ -32,11 +34,23 @@
 if not test.where_is('clang'):
     test.skip_test("Could not find 'clang', skipping test.\n")
 
+if sys.platform == 'win32':
```

Comment:
You can use:
test.IS_WINDOWS

Reply:
Sure, no problem, but grep tells me, in test/, 200-ish direct checks for win32, two uses of IS_WINDOWS.  If you want to pursue a strategy of "if you touch a test, convert it" that's fine - let's just write it down somewhere.

Reply:
Yes. If you touch it, make it better is the general policy.. improve docstrings, remove redundant code,etc..